host/xtest/xtest_1000.c

```diff
@@ -508,6 +509,169 @@ static void xtest_tee_test_1007(ADBG_Case_t *c)
 	TEEC_CloseSession(&session);
 }
 
+static void uuid_to_full_name(char *buf, size_t blen, const TEEC_UUID *uuid,
+			const char *extra_suffix)
+{
+	static const char ta_dir[] = "/lib/teetz";
```

Comment:
I didn't know that had changed, I'll change it here.
I guess I need to change my test-setup also :-)